nomos-sdp/src/state.rs

```diff
@@ -15,7 +15,7 @@ pub enum ActiveStateError {
 }
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
-pub struct ActiveState(ProviderInfo);
+pub struct ActiveState(pub ProviderInfo);
```

Comment:
What is the reason of wrapping here if the inner is public? Either is hidden and we implement `AsRef<ProviderInfo>` or we do no need the wrapping type at all.

Reply:
Good catch, I was sloppy. I just needed to initialize provider state in tests. Solved in the latest commit.